versioning/scripts/cloudbuild/main.go

```diff
@@ -188,13 +190,40 @@ func readTests(testsDir string) (tests []string) {
 				continue
 			}
 			if strings.HasSuffix(f.Name(), testJsonSuffix) || strings.HasSuffix(f.Name(), testYamlSuffix) {
-				tests = append(tests, fmt.Sprintf("/workspace/%s/%s", testsDir, f.Name()))
+				tests = append(tests, workspacePrefix+fmt.Sprintf("%s/%s", testsDir, f.Name()))
 			}
 		}
 	}
 	return
 }
 
+func filterTests(structureTests []string, functionalTests []string, version versions.Version) (outStructureTests []string, outFunctionalTests []string) {
+	included := make(map[string]bool, len(structureTests)+len(functionalTests))
+	for _, test := range append(structureTests, functionalTests...) {
+		included[test] = true
+	}
+	for _, excluded := range version.ExcludeTests {
+		if !included[workspacePrefix+excluded] {
+			log.Fatalf("No such test to exclude: %s", excluded)
```

Comment:
Because nobody will see the warning message ;) I want this to abort the build because this is likely a typo. It doesn't make sense to have an excluded test that doesn't exist.